shell/browser/atom_browser_client.cc

```diff
@@ -930,13 +930,6 @@ AtomBrowserClient::GetNetworkContextsParentDirectory() {
   return {user_data_dir};
 }
 
-bool AtomBrowserClient::ShouldBypassCORB(int render_process_id) const {
-  // This is called on the network thread.
-  base::AutoLock auto_lock(process_preferences_lock_);
```

Comment:
`process_preferences_lock_` can now be removed, this was the only method that read the preferences on IO thread.

Also we should now disable the corb feature using `ContentBrowserClient::CreateURLLoaderFactoryForNetworkRequests` when `web_security` is disabled.

Reply:
done :)